{{cookiecutter.project_shortname}}/README.md

```diff
@@ -4,9 +4,14 @@ Welcome to your InvenioILS instance.
 
 ## Getting started
 
-Run the `./setup.sh` script in order to setup your new InvenioILS instance.
+In order to setup your new InvenioILS instance, run the setup script::
```

Comment:
does it make sense to replace this whole file with the link to the docs? Otherwise we need to maintain it in two places